config-cmd/help.go

```diff
@@ -0,0 +1,98 @@
+package configCmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+
+	"github.com/spf13/cobra"
+	"github.com/taskcluster/taskcluster-cli/config"
+)
+
+func init() {
+	Command.AddCommand(&cobra.Command{
+		Use:   "help [<option>]",
+		Short: "Get help for TaskCluster's configuration options.",
+		RunE:  cmdHelp,
+	})
+}
+
+func cmdHelp(cmd *cobra.Command, args []string) error {
+	// default, print help for all commands
+	if len(args) == 0 {
+		printHelp(cmd)
+		return nil
+	}
+
+	// otherwise, print for one specific option
+	err := printOptionHelp(cmd, args[0])
+	if err != nil {
```

Comment:
nit: 'if err != nil { return err }; return nil' can be simplified to 'return err' (gosimple)